packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -492,13 +492,40 @@ mixin WidgetsBinding
       _debugShowWidgetInspectorOverrideNotifierObject ??= ValueNotifier<bool>(false);
   ValueNotifier<bool>? _debugShowWidgetInspectorOverrideNotifierObject;
 
+  /// The default behavior for taps in the app when the widget inspector is
+  /// enabled.
+  ///
+  /// - If true, taps in the app are intercepted by the widget inspector.
+  /// - If false, taps in the app are not intercepted by the widget inspector.
+  bool get debugWidgetInspectorDefaultTapBehaviorEnabled {
+    return debugWidgetInspectorDefaultTapBehaviorEnabledNotifier.value;
+  }
+
+  /// Sets the default behavior for taps in the app when the widget inspector is
+  /// enabled.
+  ///
+  /// See [debugWidgetInspectorDefaultTapBehaviorEnabled].
+  set debugWidgetInspectorDefaultTapBehaviorEnabled(bool value) {
+    debugWidgetInspectorDefaultTapBehaviorEnabledNotifier.value = value;
+  }
+
+  /// The notifier for the default behavior for taps in the app when the widget
+  /// inspector is enabled.
+  ///
+  /// See [debugWidgetInspectorDefaultTapBehaviorEnabled].
+  ValueNotifier<bool> get debugWidgetInspectorDefaultTapBehaviorEnabledNotifier =>
```

Comment:
why is this public if we have a public getter and setter for the value?

Reply:
Because we need to subscribe to the notifier for change updates - I've removed the getter and setter for the values so we only have the notifier